public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataAlertingTab.test.tsx

```diff
@@ -237,7 +237,7 @@ describe('PanelAlertTabContent', () => {
       }),
     ];
 
-    renderAlertTab(dashboard);
+    renderAlertTab(dashboard, dashboard);
```

Comment:
I am confused by this test, are we saying that the `dashboard`  is `DashboardModel`, and `DashboardDataDTO`?

Reply:
yeah, have a look at the dash definition. It's  just plain dashboard json